src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -194,6 +200,7 @@
   },
   {
     "id": "TA-000007",
+    "name": "AppServiceAPIApp.ManagedIdentity",
```

Comment:
I didn't put "Use" because of how long AppServiceFunctionApp.UseManagedIdentity ends up being D: Do you still want to add the Use?

Reply:
Oh, I see. We are already at 37 chars, so I think adding 3 more is ok. 
The action word helps me understand what the rule wants without having to read the description
It can also be `AppServiceAPIAPP.UseMI`?